native/packages/react-native-bpk-component-banner-alert/src/__snapshots__/AnimateAndFade-test.android.js.snap

```diff
@@ -79,11 +78,10 @@ exports[`Android AnimateAndFade should render correctly with animateOnEnter 1`]
 exports[`Android AnimateAndFade should render correctly with animateOnLeave 1`] = `
 <View
   collapsable={undefined}
-  shown={true}
   style={
     Object {
-      "height": 0.01,
-      "opacity": 0,
+      "height": null,
```

Comment:
this does not look correct, should it not be an integer?

Reply:
Actually it shouldn't! When the banner alert is expanded, a height is not rendered. This is so that if the content changes inside the height is allowed to expand / collapse accordingly. If we had a hard coded height the contents would be clipped or too short for the height.